### ISSUE TITLE
508-defect-2 [COGNITION]: Heading level should only increase by one

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#18627]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Josh_

## User Story or Problem Statement

As a screen reader user, I would like to hear headings in a proper nesting order according to their importance. When I hear heading levels skipped, it makes me wonder if I missed some important content.

## Details

The axe browser plugin is reporting a heading level being nested improperly. Screen reader users overwhelmingly use headings to navigate through pages, so nesting these appropriately has a big human impact. I've included a screenshot with the heading levels noted and recommend the team check out a [Chrome plugin called headingsMap](https://chrome.google.com/webstore/detail/headingsmap/flbjommegcjonpdmenkdiocclhjacmbi?hl=en) to see the same left pane I'm showing in the screen shots.

## Acceptance Criteria

- [ ] Heading level best practice issues no longer appear in the axe browser plugin (footer H4 best practice items may still appear, but that's an issue outside the scope of this ticket)
- [ ] No visual changes noted
- [ ] Screen readers announce heading level nesting appropriately

## WCAG or Vendor Guidance (optional)

* https://dequeuniversity.com/rules/axe/4.0/heading-order

## Screenshots or Trace Logs

<img width="1792" alt="Screen Shot 2021-01-15 at 11 29 50 AM" src="https://user-images.githubusercontent.com/14154792/104752779-1b65e080-5725-11eb-8a40-279a2b68e1a6.png">

**Comment by DanielleThierryUSDSVA** [comment]
@youngfreezyVA Quick note: Since this is in an alert component, it would likely need to be a sitewide change. We've run into these kinds of issues before. Nick or Kelson would be great people to talk to about how to best handle. Thanks!

**Comment by kelsonic** [comment]
Hey folks! This is indeed a sitewide issue. Hey @joshkimux, would you recommend just changing it to an `h2` but keeping the `h3` styling? That should be easy enough if we put the class `vads-u-font-size--h3` ...

Wait a sec! Ignore the above! I just saw that there's a new prop for `AlertBox` [here](https://github.com/department-of-veterans-affairs/component-library/blob/master/src/components/AlertBox/AlertBox.jsx#L74) called `level`, so just put in a prop called `level={2}` and see if that does the trick! Hopefully it doesn't affect the styling! 🙂

**Comment by joshkimux** [comment]
@kelsonic Yes! This is awesome! 🥳
